client/pom.xml

```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2016, 2017 Iotracks, Inc.
+    Copyright (c) 2018 Edgeworx, Inc.
```

Comment:
ioFog????

Reply:
ANd really should we change year here?

Reply:
2018 Edgeworx. I got this copyright from Kilton